frontend/apps/reader/modules/readerview.lua

```diff
@@ -782,6 +782,8 @@ function ReaderView:onReadSettings(config)
     self.page_scroll = page_scroll == 1 and true or false
     self.highlight.saved = config:readSetting("highlight") or {}
     self.page_overlap_style = config:readSetting("page_overlap_style") or G_reader_settings:readSetting("page_overlap_style") or "dim"
+    self.page_gap.height = config:readSetting("kopt_page_gap") or 8
+    self.page_gap_width = config:readSetting("kopt_page_gap") or 8
```

Comment:
page gap is already defined here:
https://github.com/koreader/koreader/blob/7fb2db8f02897b8ff27f529f752f9a6150ca9939/frontend/apps/reader/modules/readerview.lua#L53-L59

so you might want to change this to:
```diff
-   self.page_gap.height = config:readSetting("kopt_page_gap") or 8
+   self.page_gap.height = config:readSetting("kopt_page_gap") or G_reader_settings:readSetting("page_gap_width") or 8
```
and not define it on line 55.

also there it uses `scaleBySize` instead of a direct number.